src/stories/Home.stories.mdx

```diff
@@ -99,7 +99,7 @@ const dataArray = getMockDataArray('/nD_datasets/twoD');
 The library exposes a number of utility functions, which are documented below, as well as all the enums and types used by these utility functions.
 If you use only the top-level visualization components, the only utilities you need are `getDomain` and/or `useDomain`.
 
-- **`getDomain(values: number[], scaleType: ScaleType = ScaleType.Linear): Domain | undefined`** - Find the min and max values contained in a flat array of numbers. If `scaleType` is `ScaleType.Log` and the domain crosses zero, clamp the min to the first positive value or return `undefined` if the domain is not supported (i.e. `[-x, 0]`).
+- **`getDomain(values: ndarray | number[], scaleType: ScaleType = ScaleType.Linear): Domain | undefined`** - Find the min and max values contained in a array (or ndarray) of numbers. If `scaleType` is `ScaleType.Log` and the domain crosses zero, clamp the min to the first positive value or return `undefined` if the domain is not supported (i.e. `[-x, 0]`).
```

Comment:
Maybe also change the example in the previous section to pass the `dataArray` rather than the flat values?

Reply:
Good call as I saw that it was still named `findDomain`...